.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,7 +37,28 @@ This questions are the first thing we need to know to understand the context.
 - **OS** (e.g. from /etc/os-release):
 - **Kernel** (e.g. `uname -a`):
 - **Install tools**:
+  - `Please mention how/where was clsuter created like kubeadm/kops/minikube/kind etc. `
+- **Basic cluster related info**:
+  - `kubectl version`
+  - `kubectl get nodes -o wide`
```

Comment:
Do we really need users nodes information?

Reply:
host-os, possibility of daemonset, possibiity of packet-filtering between nodes etc comes in handy. If not advisable, then I will remove hostinfo once you confirm.

Reply:
nah, sounds good to keep